src/app/_directives/numbers-only.directive.spec.ts

```diff
@@ -0,0 +1,40 @@
+import { Component, DebugElement } from '@angular/core';
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+import { By } from '@angular/platform-browser';
+import { NumberDirective } from './numbers-only.directive';
+
+@Component({
+  template: `<input type="text" appNumbersOnly [numbericType]="'number'">`
```

Comment:
Can we remove [numbericType} and all tests associated with it as it is not used in the main file

Reply:
removed numbericType from spec file component template